cmd/vic-machine/common/utils_test.go

```diff
@@ -120,3 +119,66 @@ func TestCheckUnsupportedCharsDatastore(t *testing.T) {
 		t.Errorf("got %q, expected error for %q", err, test.S)
 	}
 }
+
+func TestCheckURLValidationOutputParity(t *testing.T) {
+	tests := []struct {
+		S     string
+		valid bool
+	}{
+		{"nfs://127.0.0.1:/my/data/here:labels", true},
```

Comment:
If there is no label the check before this function is called will fail the validation. I will add a case for it here, since an error should occur during the url parsing. I will also add the empty string which should get caught during the url field checking.